client/app/src/App.js

```diff
@@ -181,6 +182,15 @@ class App extends React.Component {
         onDisconnect: canConnect && this.handleDisconnect,
         stageId,
         ...stageData,
+        onEnd: InteractionManager.runAfterInteractions(() =>
```

Comment:
This needs to be wrapped in a callback:
```suggestion
        onEnd: () => InteractionManager.runAfterInteractions(() =>
```